### PR TITLE
 Fixes #1242 - Adds an alert informing the discontinuation of the title and focusable API

### LINKED ISSUE #1242: IE11 - <a> element is hard to click with child lexicon-icon
We originally added pointer-events:none to lexicon icons here https://github.com/liferay/clay/commit/0bec1327688a0fdafb4cb242535d6163a654c9dc.  This makes sense as lexicon-icons shouldn't have to trigger events however in IE11 the lexicon-icon "covers" the parent element preventing it from being clickable.

https://jsfiddle.net/L430qpmc/7/

EDIT : 

- I realize my example isn't the best but to properly explain the issue : 
The _first image link_ on IE11 can only be clicked by a small sliver of the image.  The expected behavior is being able to click the image link `a` more or less wherever the red background is.  The blue background "blocks" the red from being clicked -- contrast the first image with the second image.

### PR DESCRIPTION
**Pull request by matuzalemsteles**

### CODE REVIEW

**Comment by matuzalemsteles** [comment]
Just for info: A malicious package (`flatmap-stream`) that was dependent on the `event-stream` package was removed from NPM... Just updating `yarn.lock` to make sure we're removing it...

More information:
- https://snyk.io/blog/malicious-code-found-in-npm-package-event-stream
- https://github.com/dominictarr/event-stream/issues/116

**Comment by pat270** [comment]
@jbalsas @matuzalemsteles removing the `focusable="false"` attribute re-introduces svg icons receiving focus on tab in IE11 and Edge. This is an IE specific attribute.

File: packages/clay-icon/src/ClayIcon.js [hunk 1]
```diff
@@ -26,15 +26,6 @@ ClayIcon.STATE = {
...
 	elementClasses: Config.string(),
 
-	/**
-	 * Flag to indicate if the svg is focusable or not
-	 * @default false
-	 * @instance
-	 * @memberof ClayIcon
-	 * @type {?bool}
-	 */
-	focusable: Config.bool().value(false),
```
Comment (pat270) [inline packages/clay-icon/src/ClayIcon.js]: See my comment below.

**Comment by jbalsas** [comment]
Got it, we can leave that then. Does the rest look good to you @pat270?

**Comment by pat270** [comment]
LGTM. Can we roll back https://github.com/liferay/clay/pull/1307 and do the same there?

**Pull request merged**